server/src/Api/Controllers/FlashcardsController.cs

```diff
@@ -53,4 +55,17 @@ public IActionResult DeleteFlashcard(int flashcardId)
         var isSuccess = _flashcardService.DeleteFlashcard(flashcardId);
         return isSuccess ? Ok() : NotFound();
     }
+
+    [HttpPost]
+    [Route("import-csv")]
```

Comment:
```suggestion
    [Route("csv")]
```

Maybe import is not necessary, since it's a post request, idk